src/webrtc.ts

```diff
@@ -81,7 +81,7 @@ export class WebrtcExtension extends EventEmitter implements StreamingClientExte
   private stanzaInstance?: NamedAgent;
 
   get jid (): string | undefined {
-    return this.client.config.jid;
+    return this.stanzaInstance?.jid;
```

Comment:
this was causing us issues when sending some of the stanzas because this wasn't always defined.